src/imfilter_cuda.jl

```diff
@@ -0,0 +1,41 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENCE in the project root.
+# ------------------------------------------------------------------
+
+function imfilter_resource(resource::CUDALibs{R},
+                         img::AbstractArray{T,N},
+                         kern::AbstractArray{K,N}) where {R, T<:Real,K<:Real,N}
```

Comment:
Please remove type annotations as suggested in the previous round of reviews.

Reply:
The type annotation of resource is necessary for multiple dispatching. The type annotations of img and kern are inherited from the original **imfilter_gpu** function. Based on these observations, do you confirm the need of removing these type annotations.